all_in_one_seo_pack.php

```diff
@@ -417,6 +417,10 @@ function aioseop_init_class() {
 		require_once( AIOSEOP_PLUGIN_DIR . 'admin/display/menu.php' );
 		require_once( AIOSEOP_PLUGIN_DIR . 'admin/class-aioseop-notices.php' );
 
+		if ( AIOSEOPPRO ) {
+			require_once( AIOSEOP_PLUGIN_DIR . 'inc/aioseop_taxonomy_functions.php' );
```

Comment:
If this is for Pro only, would this be better in `pro/class-aio-pro-init.php`?

Reply:
as above